src/DynamoPackages/PackageLoader.cs

```diff
@@ -315,10 +315,12 @@ private void TryLoadPackageIntoLibrary(Package package)
                 }
 
                 package.SetAsLoaded();
-                PackgeLoaded?.Invoke(package);
+
 
                 PythonServices.PythonEngineManager.Instance.
                     LoadPythonEngine(package.LoadedAssemblies.Select(x => x.Assembly));
+
+                PackgeLoaded?.Invoke(package);
```

Comment:
What would be the difference here to execute this slightly later?

Reply:
There is a [piece of code](https://github.com/DynamoDS/Dynamo/blob/master/src/WorkspaceDependencyViewExtension/WorkspaceDependencyView.xaml.cs#L174) in the WorkspaceDepedencyView that checks the loaded python packages. 

The code was interpreting some packages as missing because, when it was hitting there, they were not included in the loaded assemblies. So, we need to trigger the `PackgeLoaded `action after the assembly is found in the available engines.